firmware/quintuna/RSM/src/io/io_leds.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "io_led.h"
+#include "hw_gpios.h"
+
+namespace io::leds
+{
+
+void brake_light_set(bool val);
+
+const io::binary_led brake_light{ hw::gpio::brake_light_en_pin };
+const io::binary_led led{ hw::gpio::led_pin };
```

Comment:
please extern these and create a io_leds.c file to declare them.

not your fault, I implemented like this thinking we were going to get hardware level mocks